CHANGES.rst

```diff
@@ -1,6 +1,33 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-19)
+--------------------------
+
+- Improves command line tool documentation by grouping of commands and
+  command examples.
+- Centralises the way workflow name is passed to CLI commands.
+- Renames ``workflows`` to ``list`` and ``list`` to ``ls``.
+- Introduces ``du`` command for checking workspace disk usage.
+- Introduces ``mv`` command for moving files within workspace.
+- Introduces ``diff`` command for comparing two workflows.
+- Introduces ``rm`` command for removing files within workspace.
+- Introduces ``stop`` command for stopping a running workflow.
+- Introduces ``delete`` command for deleting a workflow.
+- Introduces ``run`` command which wraps ``create``, ``upload`` and ``start``
+  commands.
+- Introduces commands for opening, listing and closing interactive sessions.
+- Introduces ``--parameter`` flag for overriding workflow input parameters.
+- Introduces ``--option`` flag for passing operational options to ``start`` or
+  ``run`` commands.
+- Introduces ``--format`` flag for ``ls`` command.
+- Introduces ``--json`` flag for ``logs`` command.
+- Introduces single variable local proxy instead of decorator
```

Comment:
I think this information is kind of only for us, just as https://github.com/reanahub/reana-client/pull/282/files#r277594472, I would either group the two at the end or just not mention them, WDYT?